data/presets/highway/path/traffic_island_shared.json

```
@@ -0,0 +1,31 @@
+{
+    "locationSet": {
+        "include": [
+            "fr",
+            "lt",
+            "pl",
+            "de",
+            "il",
+            "ps"
+        ]
+    },
+    "icon": "temaki-pedestrian_and_cyclist",
+    "geometry": [
+        "line"
+    ],
+    "tags": {
+        "path": "traffic_island"
```

Same as above; taken from https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/highway/path/bicycle_foot.json#L19-L23

```suggestion
        "path": "traffic_island",
        "foot": "designated",
        "bicycle": "designated"
```